src/main/java/pages/LoginPage.java

```diff
@@ -92,26 +105,113 @@ public String getForgotYourPasswordLinkText() {
-        return Color.fromString(buttonColorFontRgba).asHex();
+        return getColorAsHex(driver, signInButton, "color");
+    }
+
+    public boolean isPageHeaderDisplayed() {
+        return driver.findElement(pageHeader).isDisplayed();
+    }
+
+    public boolean isPageNavigationBarDisplayed() {
+        return driver.findElement(pageNavigationBar).isDisplayed();
+    }
+
+    public boolean isPageFooterDisplayed() {
+        return driver.findElement(pageFooter).isDisplayed();
+    }
+
+    public boolean isNewCustomerContainerDisplayed() {
+        return driver.findElement(newCustomerContainer).isDisplayed();
+    }
+
+    public boolean isTryDemoCustomerAccessSectionDisplayed() {
+        return driver.findElement(tryDemoCustomerAccessSection).isDisplayed();
+    }
+
+    public boolean isNewCustomersTitleDisplayed() {
+        return driver.findElement(newCustomersTitle).isDisplayed();
+    }
+
+    public String getNewCustomersTitleText() {
+        return driver.findElement(newCustomersTitle).getText();
+    }
+
+    public boolean isNewCustomersMainTextDisplayed() {
+        return driver.findElement(newCustomersMainText).isDisplayed();
+    }
+
+    public String getNewCustomersMainText() {
+        return driver.findElement(newCustomersMainText).getText();
+    }
+
+    public boolean isCreateAccountButtonEnabled() {
+        return driver.findElement(createAccountButton).isDisplayed();
+    }
+
+    public String getCreateAccountButtonText() {
+        return driver.findElement(createAccountButton).getText();
+    }
+
+    public String getCreateAccountButtonLink() {
+        return driver.findElement(createAccountButton).getAttribute("href");
+    }
+
+    public boolean isTryDemoCustomerAccessTitleDisplayed() {
+        return driver.findElement(tryDemoCustomerAccessTitle).isDisplayed();
+    }
+
+    public String getTryDemoCustomerAccessTitleText() {
+        return driver.findElement(tryDemoCustomerAccessTitle).getText();
+    }
+
+    public boolean isTryDemoCustomerAccessEmailDisplayed() {
+        return driver.findElement(tryDemoCustomerAccessEmail).isDisplayed();
+    }
+
+    public String getTryDemoCustomerAccessEmailText() {
+        return driver.findElement(tryDemoCustomerAccessEmail).getText();
+    }
+
+    public boolean isTryDemoCustomerAccessPasswordDisplayed() {
+        return driver.findElement(tryDemoCustomerAccessPassword).isDisplayed();
+    }
+
+    public String getTryDemoCustomerAccessPasswordText() {
+        return driver.findElement(tryDemoCustomerAccessPassword).getText();
+    }
+
+    public String getCreateAccountButtonFontColorAsHex() {
+        return getColorAsHex(driver, createAccountButton, "color");
+    }
+
+    public String getCreateAccountButtonBackgroundColorAsHex() {
+        waitUntilElementsIsPresented(driver, createAccountButton, 5);
+        return getColorAsHex(driver, createAccountButton, "background-color");
```

Comment:
"background-color", "color", "href" strings are repeated many times in the class. 
For cases like that class constants should be created and reused:
private static final String BACKGROUND_COLOR_PROPERTY = "background-color";
private static final String COLOR_PROPERTY = "color";
private static final String HREF_ATTRIBUTE = "href";

Reply:
If the strings will be reused in different class, than I would suggest to create 2 enums:
1. CssProperty with enum BACKGROUND_COLOR and COLOR
2. Attribute with enum HREF.

Then you will be use them in different classes anytime you need to get specific css property or attribute. And you will be able to extends these enum classes and add new properties or arguments anytime you need.